docs/testing/manual_testing/QR Scan.md

```diff
@@ -0,0 +1,34 @@
+Note: To fully test the QR Scan screen and its logic, it's necessary to test it on a real Android device
```

Comment:
We might consider a followup issue with additional instructions on setting up a fixed image as the camera input on the emulator.  The Android emulator supports it, but it requires manual config when creating the AVD.  (or digging into the emulator settings).

I don't think we can make it programmatic though, unless the gradle managed devices feature coming in the next version of AGP allows it.

Reply:
Yep, I can prepare the emulator test instructions now.